crest/Assets/Crest/Crest/Shaders/OceanShaderHelpers.hlsl

```diff
@@ -40,4 +56,24 @@ float CrestLinearEyeDepth(const float i_rawDepth)
 #endif // _PROJECTION
 }
 
+// Works for all pipelines.
+float CrestMultiSampleSceneDepth(const float i_rawDepth, const float2 i_positionNDC)
+{
+	float rawDepth = i_rawDepth;
+
+	if (_CrestDepthTextureOffset > 0)
+	{
+		// We could use screen size instead.
+		float2 texelSize = _CameraDepthTexture_TexelSize.xy;
+		int3 offset = int3(-_CrestDepthTextureOffset, 0, _CrestDepthTextureOffset);
```

Comment:
If the offset is destined to be used as a float (exclusively), it may be worth making it all floats. Having said that I was unable to find out how much type conversion costs on GPU so hard to know if it matters. Will go with your judgement.

Reply:
I'll be using it with Load too (maybe exclusively) once it is in SRP.

Reply:
Ok cool